### PR TITLE
Add product logos to MVP home page

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Add product logos to MVP home page.

## Significant changes and points to review

- Adds logos to the product list just below the hero on the home page.

## Issue / Bugzilla link

#14892

### CODE REVIEW

**Review by reemhamz — APPROVED** [review]
Looking good, however the icons are currently _really tiny_ and should be resized to be bigger, both for users + to match the design:

Currently: 
<kbd>Screenshot 2024-09-13 at 12 16 21 PM</kbd>

Design:
<kbd>Screenshot 2024-09-13 at 12 16 47 PM</kbd>

So once that's resized, it should be good to merge. Otherwise, r+wc

File: bedrock/mozorg/templates/mozorg/home/includes/m24/products.html [hunk 1]
```diff
@@ -11,37 +11,37 @@ <h2 class="m24-c-copy-title">Surf the web, on your terms</h2>
...
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://monitor.mozilla.org/{{ utm_params }}" data-cta-text="Monitor" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-monitor" href="https://monitor.mozilla.org/{{ utm_params }}" data-cta-text="Monitor" data-cta-position="product-list">
         <strong class="m24-c-spring-title">Monitor</strong>
         <span class="m24-c-spring-info">Check if your data has been breached then make it private.</span>
       </a>
     </li>
     <li class="m24-c-spring-item">
-      <a class="m24-c-spring-link" href="https://www.thunderbird.net/{{ utm_params }}" data-cta-text="Thunderbird" data-cta-position="product-list">
+      <a class="m24-c-spring-link m24-t-product-thunderbird" href="https://www.thunderbird.net/{{ utm_params }}" data-cta-text="Thunderbird" data-cta-position="product-list">
```
Comment (reemhamz) [inline bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:44]: **Nit:** I also noticed the design uses a different Thunderbird image?

Screenshot 2024-09-13 at 12 19 50 PM

Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:44]: This one seems wrong and broken to me?

**Comment by stephaniehobson** [comment]
@reemhamz Bah, they changed the design. Can we merge this because it's not broken and I'll revisit the code to resize the logos later?

File: bedrock/mozorg/templates/mozorg/home/includes/m24/products.html (hunk 1 above)
Comment (reemhamz) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:44]: Yeah it looked like a variation I haven't seen before

**Comment by reemhamz** [comment]
Sure :)

File: bedrock/mozorg/templates/mozorg/home/includes/m24/products.html (hunk 1 above)
Comment (janbrasna) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:44]: The current TB logo design SVG uses gradients and filters that a) often break on edits, leaving weird artifacts, b) trigger known bugs in some image libraries, messing up the colors… 

So seeing this "blackbird" has some kind of an unusual element appear down over the envelope, this is most likely a broken rendering of the logo SVG, after cropping or resizing it and working with it within a bigger document.

**Pull request merged**